src/irmin/irmin.ml

```diff
@@ -54,6 +54,54 @@ module Path = struct
   module type S = S.PATH
 end
 
+exception Closed
```

Comment:
no need for this exception any more

Reply:
this can be raised by the `Make_ext` functor now! (not by the backends)